src/components/PlayerAutocomplete/PlayerAutocomplete.tsx

```diff
@@ -124,7 +124,8 @@ export class PlayerAutocomplete extends React.PureComponent<PlayerAutocompletePr
                 }
             })
             .catch((err) => {
-                console.log(err);
```

Comment:
Haha I deleted this on my local environment too. Console was very red.

Since aborting requests in flight is part of a normal user journey, I think suppressing the error is reasonable. Also, doesn't `requests.ts` post a duplicate error, with more info if it's not an abort?